src/objects.js

```diff
@@ -2,38 +2,43 @@
 
 // @TODO replace with actual methods
 // as we did it at other files
-import { basename, parse, extname } from 'path';
-import { fixPath, readAllFiles } from './utils';
```

Comment:
can you explain why line that importing from utils was deleted?
i'm ok with that if this line was not necessary - not clear as well